### ISSUE TITLE
Segfault 11 or ivm::js_error when making big strings in isolate

### ISSUE THREAD

**Issue by AgentME** [laverdet/isolated-vm#12]
[This code](https://gist.github.com/AgentME/0c5f6bd2234ec3ca6799b0cd1f0ceaf2) (based on the memory-eating example) usually triggers one of the following two errors: 

```
libc++abi.dylib: terminating with uncaught exception of type ivm::js_error<&(v8::Exception::Error(v8::Local<v8::String>))>: std::exception
Abort trap: 6
```

or

```
Segmentation fault: 11
```

**Comment by laverdet** [comment]
Thanks for the report! Please try again with a fresh copy from github and let me know if that works for you.

**Comment by AgentME** [comment]
That fixed the issue! However, the catch callback is called with the value `null`; is that intended?

**Comment by jeromegn** [comment]
I ran across the same thing and using master also fixed it for me.

Time for a release?

**Comment by laverdet** [comment]
I pushed 0.1.3 for you with current master (including the fix for #23) :)

**Comment by jeromegn** [comment]
Thanks a lot!
